package.json

```diff
@@ -9,7 +9,6 @@
     "@fortawesome/react-fontawesome": "^0.1.18",
     "@reduxjs/toolkit": "^1.8.1",
     "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^13.1.1",
```

Comment:
Why is it deleted? Are you sure about that?

Reply:
I dont know how it was deleted...